.editorconfig

```diff
@@ -0,0 +1,13 @@
+# @see: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Global styles
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
```

Comment:
would you mind 4-space indentation instead? I lose my sense of indentation with just 2 spaces.